user/super/com/google/gwt/emul/java/lang/annotation/AnnotationTypeMismatchException.java

```diff
@@ -1,25 +0,0 @@
-/*
```

Comment:
This one is not an annotation, but 
1) there is no use-case for throwing this from browser code
2) [both methods declared by this class](https://docs.oracle.com/en/java/javase/17/docs/api//java.base/java/lang/annotation/AnnotationTypeMismatchException.html) were missing from emulation

Probably better to remove than add the missing methods.

Reply:
That does seem like a weird one to be emulated - looks like it could have been a part of a push to emulate anything that can be emulated way back: https://github.com/gwtproject/gwt/commit/21eef3b0834e1963ba6d519ffb0c9a889c2dd5b9

That would make it at least possible to handle such exceptions in gwt-rpc clients.

However in this case, since the Method can't be made into a field, without a custom field serializer, this can't work.

Normally I'd be inclined to say we shouldn't take away existing emulation that someone _might_ be using... but I think this one is pretty sketchy to begin with.